hypervisor/common/hypercall.c

```diff
@@ -995,3 +995,41 @@ int32_t hcall_get_cpu_pm_state(struct vm *vm, uint64_t cmd, uint64_t param)
 
 	}
 }
+
+int32_t hcall_vm_intr_monitor(struct vm *vm, uint16_t vmid, uint64_t param)
+{
+	struct acrn_intr_monitor *intr_hdr;
+	uint64_t hpa;
+	struct vm *target_vm = get_vm_from_vmid(vmid);
+
+	if (!is_vm0(vm) || (target_vm == NULL)) {
```

Comment:
Hi Minggui, we don't need to check whether it is vm0 or not. Please remove it and add related precondition, please refer to commit bb0a2bc, thanks!

Reply:
got it, I'm modifying it.